app/src/main/java/com/example/bisit/ui/myPageOwner/DialogAddCoupon.kt

```
@@ -0,0 +1,195 @@
+            }
+            
+            val today = Calendar.getInstance().apply {
+                set(Calendar.HOUR_OF_DAY, 0)
+                set(Calendar.MINUTE, 0)
+                set(Calendar.SECOND, 0)
+                set(Calendar.MILLISECOND, 0)
+            }
+            
+            val diff = expiryCalendar.timeInMillis - today.timeInMillis
+            (diff / (24 * 60 * 60 * 1000)).toInt()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)